docs/spec/qualifiers.rst

```diff
@@ -213,41 +213,52 @@ following should be allowed::
 Syntax
 ^^^^^^
 
-``Annotated`` is parameterized with a type and an arbitrary list of
-Python values that represent the annotations. Here are the specific
-details of the syntax:
+``Annotated`` is parameterized with a *base expression* and an arbitrary list of
```

Comment:
```suggestion
``Annotated`` is parameterized with a *base expression* and an arbitrary number of
```

Though "arbitrary" isn't quite right either, in fact, since there has to be at least one metadata element. (But that's stated explicitly lower down, so maybe it's okay to have slightly imprecise language here.)

Anyway, you don't have to deal with these nits now if you don't want to, since the issues pre-exist this PR

Reply:
How is this new wording? 

> Anyway, you don't have to deal with these nits now if you don't want to, since the issues pre-exist this PR

I ended up rewriting most of the Annotated section so I consider any additional clarity issues to be in scope.

Reply:
> How is this new wording?

Much improved, thanks!